src/components/Chat/AnonymousInputMessage.tsx

```diff
@@ -97,25 +218,52 @@ const AnonymousInputMessage = ({onSendButtonClicked, type}: AnonymousInputMessag
     return colors.bondi_blue;
   }, [isDisableButton()]);
 
+  const plusButtonStyle = React.useCallback(() => {
+    if (type === 'SIGNED') return colors.darkBlue;
+    return colors.bondi_blue;
+  }, []);
+
   return (
     <>
       <View style={styles.container}>
         <View style={styles.containerInput}>
-          <TextInput multiline style={styles.input} onChangeText={onChangeInput} value={text} />
-          <TouchableOpacity
-            style={styles.btn}
-            disabled={isDisableButton()}
-            onPress={handleSendMessage}>
-            <IconSend
-              style={styles.icSendButton}
-              fillBackground={sendButtonStyle()}
-              fillIcon={colors.white}
-            />
-          </TouchableOpacity>
+          <TextInput
+            multiline
+            style={styles.input}
+            onChangeText={onChangeInput}
+            value={text?.trim()}
+          />
+          {text?.trim() !== '' && (
+            <TouchableOpacity
+              style={styles.btn}
+              disabled={isDisableButton()}
+              onPress={handleSendMessage}>
+              <IconSend
+                style={styles.icSendButton}
+                fillBackground={sendButtonStyle()}
+                fillIcon={colors.white}
+              />
+            </TouchableOpacity>
+          )}
+          {text?.trim() === '' && (
+            <TouchableOpacity style={styles.btn} onPress={onSelectAttachment}>
+              <IconPlusAttachment style={styles.icSendButton} fillIcon={plusButtonStyle()} />
+            </TouchableOpacity>
+          )}
         </View>
       </View>
 
-      <SheetEmoji ref={refEmoji} selectEmoji={onSelectEmoji} />
+      <BottomSheetAttachment
+        ref={refAttachment}
+        onOpenMedia={onOpenMedia}
+        onOpenGIF={onSelectEmoji}
+        onOpenCamera={onOpenCamera}
+        onOpenFile={onOpenFile}
+        isLoadingUploadMedia={isLoadingUploadImageMedia}
+        isLoadingUploadGIF={false}
+        isLoadingUploadCamera={isLoadingUploadImageCamera}
+        isLoadingUploadFile={false}
+      />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The component `BottomSheetAttachment` has several new props related to loading states. Ensure that these props are being used correctly within the component and that their state is managed properly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->